guides/quick-start.md

```diff
@@ -6,43 +6,57 @@ In this guide, we’ll set up a full local instance of the Open Commerce platfor
 
 ## What you’ll need
 
-- [Docker](https://www.docker.com/)
 
+- We recommend installing [nvm](https://github.com/nvm-sh/nvm)
+- [14.18.1 ≤ Node version < 16](https://nodejs.org/ja/blog/release/v14.18.1/)
+- [Git](https://git-scm.com/)
+- [Yarn](https://yarnpkg.com/cli/install) (for the storefront)
+- [Docker](https://www.docker.com/)
 - [Docker Compose](https://docs.docker.com/compose/)
 
 - Windows users: [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and [Docker for WSL](https://docs.docker.com/docker-for-windows/wsl/)
 
-## Clone and start the platform
-
-The simplest way to get an instance up and running on your local machine is by using the [development platform](https://github.com/reactioncommerce/reaction-development-platform), a set of `make` scripts that coordinates all of the different pieces of the Open Commerce software system.
-
-To set up the Open Commerce development platform, navigate to a Unix or WSL directory and run:
-
-```bash
-git clone https://github.com/reactioncommerce/reaction-development-platform.git
-cd reaction-development-platform
-make
-```
-
-> **Note**: `make` starts containers in production mode, so you won't be able to make changes to the code and see them. For running in development mode, use `make init-dev`. See list of all commands [here](https://github.com/reactioncommerce/reaction-development-platform#project-commands)
+In addition you need to have your system setup for [SSH authentication with GitHub](`https://docs.github.com/en/authentication/connecting-to-github-with-ssh`)
 
-> **Note**: On Windows, you may need to replace `make` with `sudo make`. Then run `chown -R` on the `reaction-development-platform` directory to set your user as the owner of all the files it contains.
+## Install the CLI
 
-Behind the scenes, the `make` process clones all of the relevant Open Commerce software repositories, sets up each environment, and pulls, builds, and starts each Docker container in production mode.
+`npm install -g @reactioncommerce/reaction-cli`
```

Comment:
This installs the deprecated cli. It should be replaced with `npm uninstall -g reaction-cli`.

Reply:
This does not install the deprecated CLI, have you tested it?

Reply:
<img width="727" alt="Screenshot 2022-08-22 at 12 25 44" src="https://user-images.githubusercontent.com/32718995/185887824-041280e1-0b80-44c3-b8b9-fb91321b26c8.png">
https://www.npmjs.com/package/@reactioncommerce/reaction-cli

Reply:
```
 »» npm install -g reaction-cli
/home/brent/.nvm/versions/node/v14.18.1/bin/reaction -> /home/brent/.nvm/versions/node/v14.18.1/lib/node_modules/reaction-cli/index.js
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^2.3.2 (node_modules/reaction-cli/node_modules/jest-haste-map/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ reaction-cli@1.2.1
added 107 packages from 20 contributors and updated 461 packages in 37.566s
[brent@ferret: 17:21:32 ~/Projects/merchstack/docs  (getting-started-cli) - [2015]
 »» reaction help

Reply:
Also why would I issue the uninstall command to install the CLI?

Reply:
Ah I see, need to drop the org for some reason